Documentation/gettingstarted/hubble-install.rst

```diff
@@ -0,0 +1,40 @@
+.. tabs::
+
+   .. group-tab:: Linux
+
+      Download the latest hubble release:
+
+      .. parsed-literal::
+
+         curl -L https://github.com/cilium/hubble/releases/latest/download/hubble-linux-amd64.tar.gz | tar zxf -
+
+      and move the ``hubble`` CLI to a directory listed in the ``$PATH`` environment variable:
+
+      .. parsed-literal::
+
+         sudo mv hubble /usr/local/bin
+
+   .. group-tab:: MacOS
+
+      Download the latest hubble release:
+
+      .. parsed-literal::
+
+         curl -L https://github.com/cilium/hubble/releases/latest/download/hubble-darwin-amd64.tar.gz | tar zxf -
+
+      and move the ``hubble`` CLI to a directory listed in the ``$PATH`` environment variable:
+
+      .. parsed-literal::
+
+         sudo mv hubble /usr/local/bin
+
+   .. group-tab:: Windows
+
+      Download the latest hubble release:
+
+      .. parsed-literal::
+
+         curl -L https://github.com/cilium/hubble/releases/latest/download/hubble-windows-amd64.tar.gz
+
+      and move the ``hubble.exe`` CLI to your ``$PATH`` after extracting it from the tarball.
```

Comment:
```suggestion
      and move the ``hubble.exe`` CLI to a directory listed in the ``%PATH%`` environment variable after extracting it from the tarball.
```
On Windows, it's `%PATH%`, not `$PATH`. Not sure if we can give an example with the default `%PATH%` as well as I don't know what the common practice is.